src/librbd/CMakeLists.txt

```diff
@@ -191,6 +191,7 @@ if(WITH_RBD_RWL)
     cache/pwl/Types.cc
     cache/pwl/ReplicatedWriteLog.cc
     cache/pwl/AbstractWriteLog.cc
+    cache/pwl/PWLImageDispatch.cc
```

Comment:
Nit: no need for the `PWL` prefix in the `pwl` subdirectory/namespace

Reply:
cache/pwl/ImageDispatch.cc is better. updated.